src/main/webapp/script.js

```diff
@@ -294,13 +297,12 @@ function getGraphDisplay(graphNodes, graphEdges, mutList, reason) {
     node.tip.show();
   });
 
-  // If a node is searched, color it (it's fuchsia because I thought it was pretty, but definitely open to change! )
-  const nodeFilter = document.getElementById("node-name-filter");
-  if (nodeFilter && nodeFilter.value) {
-    const target = findNodeInGraph(cy, nodeFilter.value);
-    if (target) {
-      target.style('background-color', colorScheme["filteredNodeColor"]);
-    }
+  // Color the queried nodes (it's fuchsia because I thought it was pretty, but definitely open to change! )
```

Comment:
Right now mutation display has color changes, and highlighting nodes has border-width changes. I wonder if changing both color and boarder-width for queried nodes might confuse with other forms of highlighting? What happens if a node was both queried, and has a token change mutation applied? 

I don't really have a great suggestion either so feel free to discuss among yourselves.

Reply:
ahh yea that's definitely a great point. we'll think about it! maybe we could potentially come up with something by the time we do the task about making the frontend look nicer/improving readability?